aggregate_config:
  # filters:
  #   - {key: /hw/cpu/model, val: 158, predicate: '='}
  statistics:
    - raw_metric: time
      new_metric: avg_time
      type: frequency
